erpnext/stock/doctype/pick_list/pick_list.py

```diff
@@ -116,12 +116,15 @@ def get_items_with_location_and_quantity(item_doc, item_location_map):
 		if item_location.serial_no:
 			serial_nos = '\n'.join(item_location.serial_no[0: cint(stock_qty)])
 
+		auto_set_serial_no = frappe.db.get_single_value("Stock Settings", "automatically_set_serial_nos_based_on_fifo")
+		auto_set_batch_no = frappe.db.get_single_value("Stock Settings", "automatically_set_batch_nos_based_on_fifo")
```

Comment:
`automatically_set_batch_nos_based_on_fifo` does not exist in Stock Settings

Reply:
Ah, you're right, I'll remove this line